Sming/Arch/Esp32/Components/esp32/sdk/soc.mk

```diff
@@ -0,0 +1,10 @@
+#
+# soc
+#
+
+ifeq (esp32,$(SMING_SOC))
+    # esp_dport_access_reg_read is added as an undefined symbol because otherwise
+    # the linker can ignore dport_access.c as it would no other files depending on any symbols in it.
```

Comment:
> as it would no other files depending on any symbols in it.

Maybe you meant `as there are no other files depending on any symbols in it` ?

Reply:
Maybe. Just cut'n'pasted from the IDF :-)
I'll remove the comment, it's superfluous.